components/range_cache_memory_engine/tests/failpoints/test_memory_engine.rs

```diff
@@ -645,3 +645,101 @@ fn test_load_with_gc() {
         .unwrap_err();
     engine.snapshot(region.id, 0, range, 7, 100).unwrap();
 }
+
+#[test]
+fn test_region_split_before_batch_loading_start() {
```

Comment:
Please add some comment about this test.

Reply:
Done